### ISSUE TITLE
Change display of  item name on mobile/narrow windows

### ISSUE THREAD

**Issue by poliorcetics** [rust-lang/rust#79966]
First discussed in #79936.

After #79936, `rustdoc` will display items names as such:

![Screenshot 2020-12-12 at 16 12 36](https://user-images.githubusercontent.com/7951708/101987530-dc132280-3c94-11eb-804c-c37657d18b4b.png)

I wonder if something like this would be better in term of usability. It would add more scrolling but narrow screen, especially mobile ones, are made for scrolling:

![Screenshot 2020-12-12 at 16 13 32](https://user-images.githubusercontent.com/7951708/101987647-c3efd300-3c95-11eb-879c-7aaecc3f4931.png)

(This is a dirty photo-montage, sorry for the low quality)

@rustbot label T-rustdoc A-rustdoc-ui

**Comment by jyn514** [comment]
Duplicate of https://github.com/rust-lang/rust/issues/78799
